release-notes/8.0/preview/api-diff/preview3/Microsoft.NETCore.App/8.0-preview3_System.Net.md

```diff
@@ -0,0 +1,16 @@
+# System.Net
+
+``` diff
+ namespace System.Net {
+-    public class IPAddress {
++    public class IPAddress : IFormattable, IParsable<IPAddress>, ISpanFormattable, ISpanParsable<IPAddress> {
++        string IFormattable.ToString(string format, IFormatProvider formatProvider);
++        static IPAddress IParsable<IPAddress>.Parse(string s, IFormatProvider provider);
++        static bool IParsable<IPAddress>.TryParse([NotNullWhenAttribute(true)] string s, IFormatProvider provider, [NotNullWhenAttribute(true)] out IPAddress result);
```

Comment:
Do we know why the tool may not have included this?
```suggestion
+        static bool IParsable<IPAddress>.TryParse([NotNullWhenAttribute(true)] string? s, IFormatProvider provider, [NotNullWhenAttribute(true)] out IPAddress result);
```

Reply:
Probably a bug in Cci. I'll add it to my list of pending things to investigate.